### PR TITLE
Make kubemark setup in kubetest much cleaner

### PR DESCRIPTION
**Pull request by shyamjvs**
Much needed cleanup in the kubemark setup!

/cc @krzyzacy @fejta

### CODE REVIEW

File: kubetest/main.go [hunk 1]
```diff
@@ -734,7 +734,7 @@ func prepare(o *options) error {
 	}
 
 	switch o.provider {
-	case "gce", "gke", "kubemark", "kubernetes-anywhere", "node":
+	case "gce", "gke", "kubernetes-anywhere", "node":
```
Comment (krzyzacy) [inline kubetest/main.go]: do we plan to get rid of `--provider=kubemark`? (from log dump)

Comment (shyamjvs) [reply kubetest/main.go]: Yes.. I'll follow it up

**Pull request merged**